lib/linux_admin/network_interface/network_interface_rh.rb

```diff
@@ -0,0 +1,106 @@
+require 'ipaddr'
+require 'pathname'
+
+module LinuxAdmin
+  class NetworkInterfaceRH < NetworkInterface
+    IFACE_DIR = "/etc/sysconfig/network-scripts"
+
+    # @return [Hash<String, String>] Key value mappings in the interface file
+    attr_reader :interface_conf
+
+    # @param [String] Name of the network interface to manage
+    def initialize(interface)
+      super
+      @iface_file = Pathname.new(IFACE_DIR).join("ifcfg-#{@interface}")
+      reload
+    end
+
+    # Parses the interface configuration file into the @interface_conf hash
+    def reload
+      @interface_conf = {}
```

Comment:
Probably not considering we want to abstract as much of the config file itself as possible.